.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: Release
+on:
+  release:
+    types:
+      # works for building pre-releases
+      - published
+      # works for releases and changing from pre-release to release
+      - released
+
+env: 
+ 
+  DOCKER_IMAGE_NAME: newrelic/nri-kube-events
+  RELEASE_VERSION: ${{ github.event.release.name }}
+  S3_PATH: s3://nr-downloads-main/infrastructure_agent
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Extract version from release tag
+        run: |
+          RELEASE_VERSION="$(echo $RELEASE_VERSION | grep -oE '([0-9.]*[0-9])$')"
+          echo "RELEASE_VERSION=$RELEASE_VERSION">> $GITHUB_ENV
+          
+          DOCKER_IMAGE_NAME="$DOCKER_IMAGE_NAME:$RELEASE_VERSION"
+          echo "DOCKER_IMAGE_NAME=$DOCKER_IMAGE_NAME" >> $GITHUB_ENV
+          
+      - name: Test for prerelase
+        if: "github.event.release.prerelease"
+        run: |
+          DOCKER_IMAGE_NAME="$DOCKER_IMAGE_NAME-pre"
+          echo "DOCKER_IMAGE_NAME=$DOCKER_IMAGE_NAME" >> $GITHUB_ENV
+
+          S3_PATH="$S3_PATH/test"
+          echo "S3_PATH=$S3_PATH" >> $GITHUB_ENV
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build and push
+        run: |
+          make docker-build
```

Comment:
We might be missing to push `:latest` image here. On k8s-metadata-injection was missing and I added it this very morning: https://github.com/newrelic/k8s-metadata-injection/pull/73/commits/e393e1d3a36a1faae5bc7dd5010e629f0a61e087